modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/ResourceAdminServiceRestClient.java

```diff
@@ -0,0 +1,93 @@
+package org.wso2.identity.integration.test.restclients;
+
+import io.restassured.http.ContentType;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.wso2.carbon.automation.engine.context.beans.Tenant;
+import org.wso2.identity.integration.test.rest.api.common.RESTTestBase;
+
+import java.io.IOException;
+
+public class ResourceAdminServiceRestClient extends RESTTestBase {
```

Comment:
Does this client only handles email templates?

Reply:
Shall we call it `EmailTemplatesRestClient`

Reply:
Fixed with https://github.com/wso2/product-is/pull/15931/commits/485359d951310b8e58208bdd9efad7e106873bf6